department-of-veterans-affairs/va.gov-cms
# VACMS-2512: Update facility service config to obscure fields on node form

## Description
See #2512 

## Testing done
Visual, behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/89652720-e9a63f00-d893-11ea-9edd-5551a212735a.png)

## QA steps
As an administrator, go to `/admin/structure/types/manage/health_care_local_health_service/form-display` and visually verify that it looks like the above screenshot.
